docs/user/resources/02-logpipeline.md

```diff
@@ -164,9 +164,12 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | Parameter | Type | Description |
 | ---- | ----------- | ---- |
 | **conditions**  | \[\]object | An array of conditions describing the status of the pipeline. |
-| **conditions.&#x200b;lastTransitionTime**  | string | An array of conditions describing the status of the pipeline. |
-| **conditions.&#x200b;reason**  | string | Reason of last transition. |
-| **conditions.&#x200b;type**  | string | The possible transition types are:<br>- `Running`: The instance is ready and usable.<br>- `Pending`: The pipeline is being activated. |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | message is a human readable message indicating details about the transition. This may be an empty string. |
+| **conditions.&#x200b;observedGeneration**  | integer | observedGeneration represents the .metadata.generation that the condition was set based upon. For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date with respect to the current state of the instance. |
+| **conditions.&#x200b;reason** (required) | string | reason contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field, and whether the values are considered a guaranteed API. The value should be a CamelCase string. This field may not be empty. |
```

Comment:
This can be [ambiguous](https://english.stackexchange.com/questions/56065/must-not-or-may-not-which-is-the-most-correct); unless we defined "may not" very specifically.

"may not" = might be, might be not, both is possible.
"must not" = is not allowed to

Reply:
Also the documentation here is auto-generated from the metav1 package https://github.com/kubernetes/apimachinery/blob/v0.29.1/pkg/apis/meta/v1/types.go#L1536